src/validation.cpp

```diff
@@ -1236,13 +1236,13 @@ CAmount GetBlockSubsidy(int nHeight, const Consensus::Params& consensusParams)
             nSubsidy *= 98884;
             nSubsidy /= 100000;
         }
-    }
```

Comment:
The culprit 😅

Reply:
Also need to add workFinalBlockSubsidy in the struct [Params](https://github.com/jnschalk/digibyte/blob/develop/src/consensus/params.h#L55)

Reply:
Ah yeah thanks, I'd put that in src/chainparams.cpp, wrong place. Will update, thankyou